src/thunderbots/software/CMakeLists.txt

```diff
@@ -268,18 +269,33 @@ if (CATKIN_ENABLE_TESTING)
             ai/primitive/direct_velocity_primitive.cpp
             ai/primitive/kick_primitive.cpp
             ai/primitive/move_primitive.cpp
-            ai/primitive/primitive.cpp
+	    ai/primitive/pivot_primitive.cpp
```

Comment:
nit: indenting also looks off on these 2 lines

Reply:
those should be correct now (along with the rest of the file)